app/controllers/script_tags_controller.rb

```diff
@@ -2,7 +2,7 @@ class ScriptTagsController < ApplicationController
   PREFIX = "/"
   def show
     #rendering hmtl script for experiment
-    render html: ("<script type=\"text/javascript\" src=\"#{@prefix}/scripts/#{params[:id]}\"/>").to_s.html_safe,
+    render html: ("<script type=\"text/javascript\" src=\"#{@prefix}/scripts/#{params[:id]}?base_url=#{params[:base_url]}\"/>").to_s.html_safe,
```

Comment:
params[:id] i params[:base_url] powinny być zabezpieczone przez escapowanie HTML/JS. Jeśli są zabezpieczone wcześniej to napisz proszę gdzie. Jeśli to w innym commicie to wyjdzie pewnie przy mergowaniu.